### PR TITLE
add prometheus node exporter chart

### PR DESCRIPTION
**Pull request by gianrubio**
This is the first chart moved from prometheus-operator to this repo.

Related to #3490

### CODE REVIEW

**Comment by gtaylor** [comment]
Are we going to be breaking the node exporter out of the main `prometheus` chart?

**Comment by prydonius** [comment]
@gianrubio see my comment in https://github.com/kubernetes/charts/pull/3490#issuecomment-375633248, could you add an OWNERS file here and would you be able to join the Kubernetes org?

**Comment by gianrubio** [comment]
@prydonius I added the OWNER file and yes I would like to join the kubernetes org. What else do you need from me to sponsor my membership?

**Comment by prydonius** [comment]
@gianrubio I'm happy to sponsor you. @mattfarina @unguiculus @viglesiasce could be one of your other sponsors. You will need to put a request together stating the sponsors, and links to contributions: https://github.com/kubernetes/community/blob/master/community-membership.md#requirements. If you don't have multiple contributions yet, is there someone else you could add the OWNERS file for now? @brancz is already in the org.

**Comment by brancz** [comment]
I'm happy to sponsor @gianrubio as well! Regarding the OWNERS file, I'd prefer not be part of it as I already have more than enough things on my plate. :slightly_smiling_face:

**Comment by gianrubio** [comment]
@prydonius I'm now part of the organization thanks for sponsoring me, could you review this PR?

File: stable/prometheus-node-exporter/values.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 200m
+  #    memory: 50Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 30Mi
+
+rbac:
```
Comment (unguiculus) [inline stable/prometheus-node-exporter/values.yaml]: This is not used.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Review by unguiculus — APPROVED** [review]

**Pull request merged**